### ISSUE TITLE
fluid simulation

### ISSUE THREAD

**Issue by Youngtreelosewater** [sharpen3d/openvat#5]
I tried using Flip Fluids and the built-in fluid simulation in Blender, but both of these cache files were destroyed when converted to a VAT and played back. Is OpenVAT not compatible with VATs that can support fluid simulations like Houdini?

Moreover, I attempted to implement VAT in Unreal, but there was a shortage of corresponding materials.

**Comment by sharpen3d** [comment]
Yes so fluids WILL NOT work in this - point caches can work but this does not handle different vert counts of vert order or poly connections over time

Yep unreal I need to add more support for- you can set up your own decoder with a little work though, it’s not too bad. You can reference the geometry nodes and I’ve left helpful resources. 

Everyone uses these differently so I’ve tried to include knowhow and resources along with the encoder but there can never really be a definitive decoder, as on the surface people use vats for one shot or looping anims, these are most powerful in cases of dynamic responding environments like grass reacting to characters or Niagara sims (which you can use a point cloud with some additional work to wrap/remesh)
